rxandroid/src/main/java/io/reactivex/android/schedulers/AndroidSchedulers.java

```diff
@@ -13,20 +13,21 @@
  */
 package io.reactivex.android.schedulers;
 
+import android.annotation.TargetApi;
+import android.os.Build;
 import android.os.Handler;
 import android.os.Looper;
-
-import java.util.concurrent.Callable;
-
 import io.reactivex.Scheduler;
 import io.reactivex.android.plugins.RxAndroidPlugins;
+import java.lang.reflect.InvocationTargetException;
+import java.util.concurrent.Callable;
 
 /** Android-specific Schedulers. */
 public final class AndroidSchedulers {
 
     private static final class MainHolder {
-
-        static final Scheduler DEFAULT = new HandlerScheduler(new Handler(Looper.getMainLooper()));
+        static final Scheduler DEFAULT
+            = new HandlerScheduler(createAsyncHandler(Looper.getMainLooper()));
```

Comment:
This seems like it's going to require a major version update. The implications to the entirety of the world which uses RxAndroid are scary and a silent change in a minor release doesn't seem appropriate.

Reply:
I think we overload `from(Looper)` with a version that takes a boolean for `async` just like the `Handler` constructor no matter what.

As to getting it into `mainThread()`, you can use the `RxAndroidPlugins` to install an init callback for its creation which you can then delegate to `from(Looper.getMainLooper(), true)`. 

I think that's the minimum-viable API to at least get this into a release now for people to start testing it. And from there we can determine whether or not it's even possible to make it the default behind `mainThread()` in the next major version–or sometime in the future–and whether or not we need to expose a `vsyncMainThread()` or similar accessor to a singleton instance for code which truly needs the old behavior.

Reply:
That works too! For some reason I had it in my head that you were against having support for both. Can update. Should I add a brief mention/example in the readme with it or save that for changelog?